source/features/open-all-notifications.tsx

```diff
@@ -79,7 +79,7 @@ function addOpenAllButton() {
 	}
 }
 
-function update() {
+function update(): boolean | void {
 	const unreadCount = select.all(unreadNotificationsClass).length;
 	if (unreadCount === 0) {
 		return false;
```

Comment:
Why are you returning `false` here? It doesn't seem like the boolean is used anywhere.